### DISCUSSION TITLE
Remove 'application' and 'war' gradle plugins from default app template

### DISCUSSION THREAD

**Discussion by gsartori** [apache/grails-core#13789] [General]
I would propose to remove the following Gradle plugins from the default Grails application template:

- `application` -> It was introduced to accomodate Micronaut and it generates the `run` gradle task. On Grails 7 it is redundant with `bootRun`.
- `war` -> Deploying an application on a web container is not the modern way of deplying apps anymore. I would not include it in the default template, one can always add it if needed.

## From this

```
plugins {
        id "groovy"
        id "org.grails.grails-web"
        id "org.grails.grails-gsp"
        id "com.github.erdi.webdriver-binaries"
        id "war"
        id "idea"
        id "com.bertramlabs.asset-pipeline" version "5.0.1"
        id "application"
        id "eclipse"
}
```

## To this

```
plugins {
    id "groovy"
    id "org.grails.grails-web"
    id "org.grails.grails-gsp"
    id "com.github.erdi.webdriver-binaries"
    id "com.bertramlabs.asset-pipeline" version "5.0.1"
    id "eclipse"
    id "idea"
}
```

**Comment by gsartori** [comment]
Starting with the cleanup of the "application" plugin https://github.com/grails/grails-forge/pull/423

**Comment by jdaugherty** [comment]
We definitely know war is still a supported mechanism of deploying.  Users of tomcat seem to make heavy use of this still.  I don't think it's going to go away given that some of the core maintainers are using it.

**Reply by gsartori** [reply]
Okay, I think we can close this discussion then
